src/pages/ReferralDetailsPage.js

```diff
@@ -46,8 +46,12 @@ function ReferralDetailsPage({route, account}) {
     if (!_.includes(_.values(CONST.REFERRAL_PROGRAM.CONTENT_TYPES), contentType)) {
         contentType = CONST.REFERRAL_PROGRAM.CONTENT_TYPES.REFER_FRIEND;
     }
+
     const contentHeader = translate(`referralProgram.${contentType}.header`);
-    const contentBody = translate(`referralProgram.${contentType}.body`);
+    const contentBody = translate(`referralProgram.${contentType}.body1`);
+    const isShareCode = contentType === CONST.REFERRAL_PROGRAM.CONTENT_TYPES.SHARE_CODE;
+    const shouldShowBody2 = isShareCode;
```

Comment:
What would happen if we did `translate(`referralProgram.${contentType}.body1` and body2 didn't exist for that content type? Would the function return an empty string, or would it throw an error? 

Because if it just returns an empty string, maybe we could use the existence of `referralProgram.${contentType}.body2` to decide whether to show body2. Rather than the contentType. That way the copy could be updated without having to update the code here.

Reply:
Ah damn, ok. This approach seems good then.